dotCMS/src/main/java/com/dotmarketing/portlets/languagesmanager/business/LanguageFactoryImpl.java

```diff
@@ -374,7 +374,7 @@ protected Language getLanguage(final long id) {
 	protected List<Language> getLanguages() {
 		List<Language> languages = CacheLocator.getLanguageCache().getLanguages();
 		if(languages != null){
-			return ImmutableList.copyOf(languages);
+			return languages;
```

Comment:
just returning the immutable list out of cache rather than creating a new list and copying it.